### ISSUE TITLE
Audio and Video Block: Consider adding an accessibility notice to the autoplay option. 

### ISSUE THREAD

**Issue by kjellr** [WordPress/gutenberg#13550]
Porting this over from notes in #8297: 

> Good to see that Autoplay is untoggled by default. It could be nice to add a little note, a la colour contrast warnings, when people toggle this, to warn them to be careful autoplaying anything lest they annoy their users.

While I agree wholeheartedly that autoplay videos are frequently annoying, autoplaying is also a potential accessibility issue. As per [W3C Guidelines](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-dis-audio.html): 

> Playing audio automatically when landing on a page may affect a screen reader user's ability to find the mechanism to stop it because they navigate by listening and automatically started sounds might interfere with that navigation.

We should consider adding some sort of notice to this option. Perhaps just as a label: 

<img width="279" alt="screen shot 2019-01-28 at 2 16 13 pm" src="https://user-images.githubusercontent.com/1202812/51860439-ae729980-2307-11e9-9307-d33862c97d9c.png">

_(The copy could certainly be improved — please share any suggestions!)_

**Comment by michelleweber** [comment]
Questions:

Do you want to go as far as stating that autoplaying is not recommended?

Do you want to specify further, a la "Autoplaying can impede sight-impaired visitors from accessing your website"?

If not, what you've got there now works. I might nix the "please," but otherwise, it's brief and clear.

**Comment by kjellr** [comment]
Cool, thanks @michelleweber. I _think_ keeping what we have seems fine. It's not totally prescriptive, but it at least lets people know there's a possible issue. Removing "Please" sounds good to me too.

If anyone from the a11y team would like to weigh in too, that'd be very helpful.

**Comment by afercia** [comment]
DIscussed during extra accessibility bug-scrub on Feb. 24th, the notice would greatly help producing accessible content.

Worth noting it should be added to both the Audio and Video blocks.

**Comment by afercia** [comment]
Note: as implemented in #15575, the message is just text that appears on a page.

The message itself should be accessible, as a notice that should be announced by screen readers.

See #15293 in the accessibility audit. Although #15293 specifically refers to notices in the media views, the base principle still applies. See also the previous #9442 and #9425.

This text should either have a role=alert or be passed to a `speak()` message.  Probably worth opening a new issue, thoughts?

**Comment by kjellr** [comment]
Yeah, a new issue for that makes sense. It sounds like something that should be implemented globally for these help messages, right?

**Comment by afercia** [comment]
Yeo. Will open a new issue, sort of duplicate of #9425.
